eox_nelp/tests/utils.py

```diff
@@ -0,0 +1,48 @@
+""" Module with methods to reuse in test cases"""
+from mock import Mock
+
+
+def generate_list_mock_data(data):
```

Comment:
if you need something like this you should consider to improve the method that already exists instead of create a new one

Reply:
Yes, but I don't want it to call it from https://github.com/eduNEXT/eox-nelp/blob/master/eox_nelp/notifications/tests/test_tasks.py because the method could be extended to other modules.
And as you see it I don't use it to `_generate subsection`, I use it to generate whatever list mock like course enrollments.

Reply:
same comment

Reply:
I dont understand the same comment, one thing I can do is delete _generate_subsection and call in that file generate_list_mock_data. But that is not obtrusive?

Reply:
I mean this method does the same thing that an older method does but in a better and wide way, so you should consider to delete the old one and improve the general code not just your implementation, that help us to mantain less code

Reply:
[fa5e619](https://github.com/eduNEXT/eox-nelp/pull/44/commits/fa5e6198633321a62221fb2968560cd61553f3d8)